src/pudl/package_data/glue/respondent_id_ferc714.csv

```diff
@@ -0,0 +1,218 @@
+respondent_id_ferc714,respondent_id_ferc714_xbrl,respondent_id_714_csv,Source,Notes
```

Comment:
Is there a reason that `respondent_id_714_csv` doesn't follow the same naming convention as the other two ID columns?

Reply:
Sheer oversight - sounds like @cmgosnell already fixed this in her branch though.